validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/OwnedValidators.java

```diff
@@ -58,6 +61,19 @@ public int getValidatorCount() {
     return Optional.ofNullable(validators.get(publicKey));
   }
 
+  public List<ActiveValidator> getActiveValidators() {
+    List<ActiveValidator> activeValidatorList = new ArrayList<>();
+    Set<BLSPublicKey> validatorKeySet = validators.keySet();
+    for (BLSPublicKey blsPublicKey : validatorKeySet) {
+      getValidator(blsPublicKey)
+          .ifPresent(
+              validator ->
+                  activeValidatorList.add(
+                      new ActiveValidator(blsPublicKey, validator.isReadOnly())));
+    }
+    return activeValidatorList;
```

Comment:
```    
    return validators
        .values()
        .stream()
        .map(validator -> new ActiveValidator(validator.getPublicKey(), validator.isReadOnly()))
        .collect(Collectors.toList());
```
something like that anyway - trying to remember that this shouldn't be considered ad-hoc, so we want it fairly clean...
I think this is roughly the right direction, and probably move `ActiveValidators` to be with OwnedValidators. but this is probably also good to flag with @ajsutton just to check he's happy with `ActiveValidators` living with `OwnedValidators` which is under the `validator.client.loader` package